src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/service/cessations/BeneficialOwnerCessationService.java

```diff
@@ -0,0 +1,76 @@
+
+  public List<Cessation> beneficialOwnerCessations(OverseasEntitySubmissionDto updateSubmission) {
+    overseasEntitySubmissionDto = updateSubmission;
+    List<Cessation> cessations = new ArrayList<>();
+    cessations.addAll(returnIndividualBeneficialOwners());
+    cessations.addAll(returnCorporateEntityBeneficialOwnerCessations());
+    cessations.addAll(returnLegalPersonBeneficialOwners());
+    return cessations;
+  }
+
+  private List<IndividualBeneficialOwnerCessation> returnIndividualBeneficialOwners() {
+    var returned = overseasEntitySubmissionDto.getBeneficialOwnersIndividual();
+    return returned.stream()
+        .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getCeasedDate() != null))
+        .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getChipsReference() != null))
+        .map(this::returnIndividualBeneficialOwnerCessation)
+        .collect(Collectors.toList());
+  }
+
+  private List<LegalPersonBeneficialOwnerCessation> returnLegalPersonBeneficialOwners() {
+    var returned = overseasEntitySubmissionDto.getBeneficialOwnersGovernmentOrPublicAuthority();
+    return returned.stream()
+            .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getCeasedDate() != null))
+            .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getChipsReference() != null))
+            .map(this::returnLegalPersonBeneficialOwnerCessation)
+            .collect(Collectors.toList());
+  }
+
+  private List<CorporateEntityBeneficialOwnerCessation>
+      returnCorporateEntityBeneficialOwnerCessations() {
+    var returned = overseasEntitySubmissionDto.getBeneficialOwnersCorporate();
+    return returned.stream()
+            .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getCeasedDate() != null))
+            .filter(beneficialOwnerCessation -> (beneficialOwnerCessation.getChipsReference() != null))
+            .map(this::returnCorporateEntityBeneficialOwnerCessation)
+            .collect(Collectors.toList());
+  }
+
+  public IndividualBeneficialOwnerCessation returnIndividualBeneficialOwnerCessation(
+      BeneficialOwnerIndividualDto bo) {
+    return new IndividualBeneficialOwnerCessation(
+            bo.getChipsReference(),
+            bo.getCeasedDate(),
+            bo.getDateOfBirth(),
+            new PersonName(bo.getFirstName(), bo.getLastName()));
+  }
+
+  public CorporateEntityBeneficialOwnerCessation returnCorporateEntityBeneficialOwnerCessation(
+      BeneficialOwnerCorporateDto bo) {
+    return new CorporateEntityBeneficialOwnerCessation(
+            bo.getChipsReference(), bo.getCeasedDate(), bo.getName());
+  }
+
+  public LegalPersonBeneficialOwnerCessation returnLegalPersonBeneficialOwnerCessation(
+      BeneficialOwnerGovernmentOrPublicAuthorityDto bo) {
+    return new LegalPersonBeneficialOwnerCessation(
+            bo.getChipsReference(), bo.getCeasedDate(), bo.getName());
```

Comment:
again, ids need to come from private data. They need to be the db ids and not hashed ids.